activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+*   Active Record query cache is now evicts least recently used entries
+
+    By default it only keeps the `50` most recently used queries.
```

Comment:
Should this be 100 instead ?

Reply:
That was fixed in https://github.com/rails/rails/pull/48192